tune_sklearn/tune_basesearch.py

```diff
@@ -411,7 +436,12 @@ def score(self, X, y=None):
             float: computed score
 
         """
-        return self.scoring(self.best_estimator_, X, y)
+        self._check_is_fitted("score")
+        if self.is_multi:
+            scorer_name = self.refit
+        else:
+            scorer_name = "score"
+        return self.scoring[scorer_name](self.best_estimator_, X, y)
```

Comment:
Scoring always becomes a dictionary, regardless of if it's single or multi metric?

Reply:
Yeah because _check_multimetric_scoring always returns a dict